include/qubes-gui-protocol.h

```diff
@@ -82,8 +87,11 @@ typedef unsigned __int32 uint32_t;
 /* Minimum version for MSG_WINDOW_DUMP_ACK */
 #define QUBES_GUID_MIN_MSG_WINDOW_DUMP_ACK 0x00010007
 
+/* Minimum version for (up to) 4x sized clipboard */
+#define QUBES_GUID_MIN_CLIPBOARD_4X 0x00010008
+
 //arbitrary
-#define MAX_CLIPBOARD_SIZE 65000
+#define MAX_CLIPBOARD_SIZE 256000
```

Comment:
Since the old size is relevant for old protocol support, better to keep MAX_CLIPBOARD_SIZE unchanged and have new constant for increased size. As you can see in the other PR, not having this constant here forces you to open code it in a few places.

As for why keeping old one unchanged - it avoids mixing protocol versions (old agent built with new header file).

Reply:
This one is resolved